ingest/defaults/config.yaml

```diff
@@ -11,7 +11,7 @@ entrez_search_term: ""
 ncbi_taxon_id: ""
 
 # The list of NCBI Datasets fields to include from NCBI Datasets output
-# These need to be the mneumonics of the NCBI Datasets fields, see docs for full list of fields
+# These need to be the mnemonics of the NCBI Datasets fields, see docs for full list of fields
```

Comment:
_non-blocking_

I read this and thought, "Is _mnemonics_ even the right word here? That seems like a wrong use of the word. ISTM it should be _names_ or _monikers_ or something." And then I saw that the linked NCBI doc refers to, e.g.  `biosample-acc` as the mnemonic for the name "BioSample accession". what?? I believe it'd be more conventional to distinguish field _ids_ vs. _names_ or field _names_ vs. _titles_; field _mnemonics_ vs. _names_ is just weird NCBI!

<small>. o O ( _mpneumonics_ — how your lungs remember things :upside_down_face: )</small>

Reply:
```suggestion
# These need to be the "mnemonics" of the NCBI Datasets fields, see docs for full list of fields
```